build/Signing/Microsoft.NET.Signing.proj

```diff
@@ -25,4 +24,4 @@
     <Exec Command="$(SignTool) -nugetPackagesPath &quot;$(NuGet_Packages)&quot; -config &quot;@(SigningConfig)&quot; &quot;$(OutDir.TrimEnd('\'))&quot;"
```

Comment:
I don't understand - SigningConfig is moving from an item to a property? It should be $(SigningConfig) here then right? I don't understand why this needs to be a property instead of an item - what's broken?